Python/pystate.c

```diff
@@ -848,6 +851,7 @@ new_threadstate(PyInterpreterState *interp)
         memcpy(tstate,
                &initial._main_interpreter._initial_thread,
                sizeof(*tstate));
+        tstate->_static = false;
```

Comment:
Please add a note pointing to [`PyThreadState_INIT()`](https://github.com/python/cpython/blob/2eee9d9cd7eb1e396fa9a4af7c5fadeeafbdaa38/Include/internal/pycore_runtime_init.h#L68-L73) and indicating that fields set there should be adjusted here as appropriate.